### PR TITLE
OSD-2924: migrate billing account to configmap, configmap structure is refactored

### PR DESCRIPTION
**Pull request by boranx**

### CODE REVIEW

File: pkg/configmap/configmap.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+)
+
+// OperatorConfigMap store data for the specified configmap
+type OperatorConfigMap struct {
+	BillingAccount string `mapstructure:"billingAccount"`
+	ParentFolderID string `mapstructure:"parentFolderID"`
+}
+
+// CheckValueNotExist checks if OperatorConfigMap filled properly
+func CheckValueNotExist(configmap OperatorConfigMap) error {
```
Comment (NautiluX) [inline pkg/configmap/configmap.go]: I find the name of this function confusing it sounds like it checks if a value exists (or does not exist?), I suggest something like ValidateConfigMap, which would make clearer that it verifies all the expected values exist.

File: pkg/controller/clusterdeployment/clusterdeployment_controller.go [hunk 2]
```diff
@@ -174,9 +172,16 @@ func (r *ReconcileClusterDeployment) Reconcile(request reconcile.Request) (recon
 		return reconcile.Result{}, nil
 	}
 
-	orgParentFolderID, err := util.GetGCPParentFolderFromConfigMap(r.client, operatorNamespace, orgGcpConfigMap)
+	// Get operatorConfigMap with values, use it for getting some value
+	// If any field does not filled by GetOperatorConfigMap, check which mapping is empty and return err
```
Comment (NautiluX) [inline pkg/controller/clusterdeployment/clusterdeployment_controller.go]: In my opinion, you don't need these comments, your code is very well speaking for itself (despite the function name `CheckValueNotExist`). :)

File: pkg/configmap/configmap.go (hunk 1 above)
Comment (boranx) [reply pkg/configmap/configmap.go]: `ValidateConfigMap` looks good.

File: pkg/controller/clusterdeployment/clusterdeployment_controller.go (hunk 2 above)
Comment (boranx) [reply pkg/controller/clusterdeployment/clusterdeployment_controller.go]: haha I'll remove them

File: README.md [hunk 3]
```diff
@@ -135,8 +136,17 @@ gcloud iam service-accounts keys create key.json \
...
-# billindacount.txt contains billing ID from https://console.cloud.google.com/billing
-kubectl create secret generic gcp-project-operator --from-file=key.json=secrets/key.json --from-file=billingaccount=secrets/billingaccount.txt -n gcp-project-operator
+kubectl create secret generic gcp-project-operator --from-file=key.json=secrets/key.json -n gcp-project-operator
+
+```
+
+## GCP Operator configmap creation
+
+```bash
+export ORGPARENTFOLDERID="240634451310" # Google Cloud organization Parent Folder ID
```
Comment (mmazur) [inline README.md]:
```suggestion
export ORGPARENTFOLDERID="292670943244" # OpenShift Dedicated DEV folder
```

Comment (NautiluX) [reply README.md]: I'd prefer to use a fictive folder ID

Comment (mmazur) [reply README.md]: Sure, we can remove our actual IDs and have a separate private gdoc on how to set configure the operator to run with our gcp env.

Comment (boranx) [reply README.md]: I can remove the actual ID here but adding private gdoc may be handled in another task.

Comment (mmazur) [reply README.md]: I'll do the private gdoc.

Comment (NautiluX) [inline README.md]:
```suggestion
export ORGPARENTFOLDERID="12345678" # Google Cloud organization Parent Folder ID
```

Comment (boranx) [reply README.md]: done

**Pull request merged**